src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -1440,6 +1471,72 @@ def test_api_video_create_by_organization_admin(self):
             },
         )
 
+    def test_api_video_create_with_scheduled_date_gets_ignored(self):
+        """
+        Create video with right access.
+
+        Try to init field starting_at and property is_scheduled on creation. Data are ignored.
+        """
+        user = factories.UserFactory()
+        organization = factories.OrganizationFactory()
+        factories.OrganizationAccessFactory(
+            role=models.ADMINISTRATOR, organization=organization, user=user
+        )
+        playlist = factories.PlaylistFactory(organization=organization)
+
+        jwt_token = AccessToken()
+        jwt_token.payload["resource_id"] = str(user.id)
+        jwt_token.payload["user"] = {
+            "id": str(user.id),
+            "username": user.username,
+        }
+
+        self.assertEqual(models.Video.objects.count(), 0)
+        # try to set starting_at and is_scheduled
+        response = self.client.post(
+            "/api/videos/",
+            {
+                "lti_id": "video_one",
+                "playlist": str(playlist.id),
+                "title": "Some title",
+                "starting_at": "2100-01-01",
```

Comment:
IMO you should not work with date in string format. Use `datetime` or `timezone.now() +/- timedelta` depending the scenario you want.

```suggestion
                "starting_at": datetime(2100, 1, 1, tzinfo=pytz.utc),
```

Reply:
Yes completly right, Sam made me change it, this one is a forgotten one, thank you